### PR TITLE
[BREAK] Remove GraphQL and grant packages

### LINKED ISSUE #14959: Remove GraphQL Support
The current implementation of our GraphQL API was an experiment that for many reasons did not get much adoption. It runs on a separate port and we were never able to fully integrate into the main code. For that reason, it has added too many dependencies that are useless for the vast majority of our deployments. It is the source of annoying issues like https://github.com/RocketChat/Rocket.Chat/issues/10714 causing unnecessary load and worries. 

We believe that supporting GraphQL is still on our long term plans, but we will have to refactor the implementation with the new architecture we are designing for high scalability.

With that in mind, we are **planning to remove the current implementation** from the code base and keep the PR https://github.com/RocketChat/Rocket.Chat/pull/8158 as a reference for future development as an auxiliary Apolo base project (_probably_).

### PR DESCRIPTION
**Pull request by sampaiodiego**
Closes #14959

Docs update https://github.com/RocketChat/docs/pull/1392

### CODE REVIEW

**Comment by rodrigok** [comment]
Need to update the documentation saying that it was removed on version 2.0

File: app/graphql/server/settings.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { settings } from '../../settings';
-
-settings.addGroup('General', function() {
-	this.section('GraphQL API', function() {
```
Comment (rodrigok) [inline app/graphql/server/settings.js]: Need a migration to remove these settings

Comment (sampaiodiego) [reply app/graphql/server/settings.js]: removed in f5f2ebc26b4370038a37e6c9433ea699dce238f9

**Comment by sampaiodiego** [comment]
Docs update https://github.com/RocketChat/docs/pull/1392

**Pull request merged**